package.json

```diff
@@ -162,7 +162,7 @@
     "chance": "^1.0.13",
     "concurrently": "3.5.1",
     "dotenv": "^5.0.1",
-    "electron": "^2.0.17",
+    "electron": "1.8.8",
```

Comment:
This would break Mac version (and maybe Windows version).
A temporary fix/hack would be to conditionally downgrade `electron` for Linux in a postinstall script.

Reply:
or even better, but only for linux